dts/arm/st/stm32f0.dtsi

```diff
@@ -49,6 +49,60 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 			reg = <0x48000000 0x1800>;
+
+			gpioa: gpio@48000000 {
```

Comment:
Is there a special reason to put gpio nodes within the pinctrl node besides that it is done in linux?

This just makes the defines longer or needs some extra logic to strip off the extra "PINCTRL" label.
A gpio device may just use the pinctrl/pinmux device but can be seen independent from the pinmux/pinctrl device.